.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+    - 2.6
```

Comment:
Might I convince you to comment out the Python 2 versions and instead file a bug against the code that it doesn't work on Python 2?  Then I can fix that problem, and then uncomment those lines.